bikeshed/MetadataManager.py

```diff
@@ -19,6 +19,11 @@ def vshortname(self):
             return "{0}-{1}".format(self.shortname, self.level)
         return self.shortname
 
+    @property
+    def groupIsW3C(self):
+        return self.group in ["csswg", "dap", "fxtf", "geolocation", "houdini", "ricg", "svg",
+                              "texttracks", "web-bluetooth-cg", "webappsec", "webspecs", "whatwg"]
+
```

Comment:
I put data like this in the config.py file. Make a frozenset there with these names. I like the function still, just want the data moved. ^_^